### PR TITLE
Extend throughput benchmark with device CLI parameter

### PR DESCRIPTION
**Pull request by TolyaTalamanov**
Added device name as the second CLI parameter with default CPU value for throughput benchmark sample

### CODE REVIEW

**Review by vurusovs — APPROVED** [review]

File: samples/cpp/benchmark/throughput_benchmark/main.cpp [hunk 1]
```diff
@@ -22,8 +22,12 @@ int main(int argc, char* argv[]) {
     try {
         slog::info << "OpenVINO:" << slog::endl;
         slog::info << ov::get_openvino_version();
-        if (argc != 2) {
-            slog::info << "Usage : " << argv[0] << " <path_to_model>" << slog::endl;
+
+        std::string device_name = "CPU";
+        if (argc == 3) {
+            device_name = TSTRING2STRING(argv[2]);
```
Comment (Wovchena) [inline samples/cpp/benchmark/throughput_benchmark/main.cpp:28]:
```suggestion
            device_name = argv[2];
```
This main doesn't take tchars

File: samples/cpp/benchmark/throughput_benchmark/main.cpp
Comment (Wovchena) [inline samples/cpp/benchmark/throughput_benchmark/main.cpp:1]: Please update README. It was moved recently. As far as I understand the new location is docs\articles_en\learn_openvino\openvino_samples\cpp_sample_throughput_benchmark.md

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/cpp/benchmark/throughput_benchmark/main.cpp
Comment (Wovchena) [inline samples/cpp/benchmark/throughput_benchmark/main.cpp:1]: I guess you are not interested in python equivalent sample, but to keep them aligned, please apply the same change for Python as well

Comment (dmatveev) [reply samples/cpp/benchmark/throughput_benchmark/main.cpp:1]: Well we're actually interested in Python equivalents - so thanks for heads-up!

Comment (TolyaTalamanov) [reply samples/cpp/benchmark/throughput_benchmark/main.cpp:1]: Done

File: samples/python/benchmark/throughput_benchmark/throughput_benchmark.py [hunk 2]
```diff
@@ -38,10 +38,10 @@ def main():
     tput = {'PERFORMANCE_HINT': 'THROUGHPUT'}
 
     # Create Core and use it to compile a model.
-    # Pick a device by replacing CPU, for example MULTI:CPU(4),GPU(8).
```
Comment (Wovchena) [inline samples/python/benchmark/throughput_benchmark/throughput_benchmark.py:41]: Comments also should be aligned

File: samples/cpp/benchmark/throughput_benchmark/main.cpp
Comment (TolyaTalamanov) [reply samples/cpp/benchmark/throughput_benchmark/main.cpp:1]: Good catches! Thanks, done

File: samples/python/benchmark/throughput_benchmark/throughput_benchmark.py (hunk 2 above)
Comment (TolyaTalamanov) [reply samples/python/benchmark/throughput_benchmark/throughput_benchmark.py:41]: Done

Comment (TolyaTalamanov) [reply samples/python/benchmark/throughput_benchmark/throughput_benchmark.py:41]: Aligned

**Review by Wovchena — APPROVED** [review]

**Pull request merged**